src/script/conversation/MessageRepository.ts

```diff
@@ -1583,30 +1582,8 @@ export class MessageRepository {
    * @param countlyId Countly new ID
    */
   public async sendCountlySync(countlyId: string) {
-    const selfConversation = this.conversationState.self_conversation();
-    if (selfConversation?.isFederated()) {
-      // TODO(federation)
-      this.logger.warn('syncly not implemented for federated env');
-      return;
-    }
-    const protoDataTransfer = new DataTransfer({
-      trackingIdentifier: {
-        identifier: countlyId,
-      },
-    });
-    const genericMessage = new GenericMessage({
-      [GENERIC_MESSAGE_TYPE.DATA_TRANSFER]: protoDataTransfer,
-      messageId: createRandomUuid(),
-    });
-
-    const eventInfoEntity = new EventInfoEntity(genericMessage, selfConversation.qualifiedId);
-    try {
-      await this.sendGenericMessageToConversation(eventInfoEntity);
-      this.logger.info(`Sent countly sync message with ID ${countlyId}`);
-    } catch (error) {
```

Comment:
wondering if we should still try catch an error (in case one appears)

Reply:
Yeah same strategy for that error catching. 
Of course there could be a backend error, in which case no message sending is going to go through. 
And there could be a logic error somewhere (an undefined value not handled somewhere or something), in which case I think it's good that the error is loud and clear and bubbles up rather than being hidden in a log. 

In general I try to catch error in rather high layers of the app, not too deep, but I am open to discussions about the right place to catch errors :)